marlowe-dashboard-client/default.nix

```diff
@@ -22,11 +22,18 @@ let
     $(nix-build ../default.nix -A marlowe-dashboard.marlowe-run-backend-invoker)/bin/marlowe-dashboard-server psgenerator $generatedDir
   '';
 
-  start-backend = pkgs.writeShellScriptBin "marlowe-pab-server" ''
+  start-backend = pkgs.writeShellScriptBin "marlowe-run-server" ''
     echo "marlowe-pab-server: for development use only"
+    export NOMAD_PORT_wbe="''${NOMAD_PORT_wbe:-8090}"
+    cat > marlowe-run.json <<EOF
+    {
+      "getWbeConfig": { "_wbeHost": "localhost", "_wbePort": $NOMAD_PORT_wbe },
+      "getStaticPath": "/var/empty"
+    }
+    EOF
     (trap 'kill 0' SIGINT;
       $(nix-build ../default.nix --quiet --no-build-output -A marlowe-dashboard.marlowe-invoker)/bin/marlowe-pab --config plutus-pab.yaml webserver &
-      $(nix-build ../default.nix -A marlowe-dashboard.marlowe-run-backend-invoker)/bin/marlowe-dashboard-server webserver -c ./config.json
+      $(nix-build ../default.nix -A marlowe-dashboard.marlowe-run-backend-invoker)/bin/marlowe-dashboard-server webserver -c ./marlowe-run.json
```

Comment:
I added this a couple of weeks ago, now I updated it with the changes you made. Is there a better way to run multiple programs at once in bash/nix? Should we do something similar to launch the WBE, node and chainlink?

Reply:
@hrajchert I don't think there's a great answer to the problem of local test services currently, but one thing I like about the scripts I wrote in shell.nix is that they reuse all of the code that's used in the real deployment. So to start the backend here you'd just have to export the port env var, not create the config file yourself or do the nix-build yourself etc.

What I'd do is create a bunch of individual scripts like I did in the shell, then create an umbrella script that starts each in the background then waits for any to fail (using the `wait` builtin) and kills them all if they do.